fighthealthinsurance/followup_emails.py

```
@@ -56,11 +60,22 @@ def dosend(
-        candidates = FollowUpSched.objects.filter(follow_up_sent=False).filter(
-            follow_up_date__lt=datetime.date.today()
-        )
-        return candidates
+    def _find_candidates(self):
+        try:
+            candidates = (
+                FollowUpSched.objects.filter(follow_up_sent=False)
+                .filter(follow_up_date__lt=datetime.date.today())
+                .distinct("email")
+            )
+            return candidates
```

Ruff (0.14.8)

Consider moving this statement to an `else` block

(TRY300)